specs/CommandBar/new-command-bar-features.md

```diff
@@ -0,0 +1,349 @@
+
+![Command bar with more button](images/more-button.jpg)
+
+When the `CommandBarFlyout` is invoked via touch, only the primary commands are shown (with the option for the user to
+expand the list of secondary commands). The developer is currently not able to control how the `CommandBarFlyout` appears when invoked, 
+and cannot control whether it can be collapsed or not. 
+
+# Description
+
+### Feature #1. Built-in styles to support adding `SplitButton` to `CommandBar` and `CommandBarFlyout`
+
+WinUI will provide built in styles for SplitButton that will allow them to conform with the look and feel of an `AppBarButton`. 
+These styles will change properties such as the `SplitButton`'s height/width, font size, color, interactions (color changes), and more. 
+
+There will be two styles provided: `CommandBarSplitButtonStyle` and `CommandBarFlyoutSplitButtonStyle`. 
+These styles can be accessed as ThemeResources in any WinUI app. 
+
+Note: if a SplitButton item inside of a CommandBar enters the CommandBar's overflow menu, it will pick up the `CommandBarFlyoutSplitButtonStyle`.
+
+For example:
+
+```xml
+<CommandBarFlyout x:Name="ImageCommandsFlyout" >  
+    <AppBarButton Icon="Copy" ToolTipService.ToolTip="Copy" Label="Copy"/>
+
+    <CommandBarFlyout.SecondaryCommands>
+        <AppBarElementContainer>
+            <SplitButton ToolTipService.ToolTip="Insert"
+                         Style="{ThemeResource SplitButtonCommandBarFlyoutStyle}">
+                <SplitButton.Content>
+                    <StackPanel Orientation="Horizontal">
+                        <TextBlock>Insert</TextBlock>
+                    </StackPanel>
+                </SplitButton.Content>
+                <SplitButton.Flyout>
+                    <MenuFlyout Placement="RightEdgeAlignedTop">
+                        <MenuFlyoutItem Text="Insert above"/>
+                        <MenuFlyoutItem Text="Insert between"/>
+                        <MenuFlyoutItem  Text="Insert below"/>
+                    </MenuFlyout>
+                </SplitButton.Flyout>
+            </SplitButton>
+        </AppBarElementContainer>
+        <AppBarButton Label="Select all"/>
+        <AppBarButton Label="Delete" Icon="Delete"/>
+    </CommandBarFlyout.SecondaryCommands>
+</CommandBarFlyout>
+```
+
+### Feature #2. `RadioMenuFlyoutSubItem`
+
+`RadioMenuFlyoutSubItem` is a new class that represents a "parent" `RadioMenuFlyout` item. 
+It is very similar to the [MenuFlyoutSubItem](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.menuflyoutsubitem?view=winrt-19041) class.
+
+All child items of a `RadioMenuFlyoutSubItem` should be of type 
+[RadioMenuFlyoutItem](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.radiomenuflyoutitem?view=winui-2.5) 
+(although this is not enforced by the API). If a child item is selected, the parent item will show as selected. 
```

Comment:
Can I at least put a separator in there?

Some people like to put dummy elements inside a flyout, e.g., to provide labels. Is that allowed?
```xml
<RadioMenuFlyoutSubItem>
    <MenuFlyoutItem IsEnabled="False">Other free options</MenuFlyoutItem>
    <RadioMenuFlyoutItem GroupName="Shipping">Enhanced shipping</RadioMenuFlyoutItem>
    <MenuFlyoutSeparator/>
    <MenuFlyoutItem IsEnabled="False">Extra charge</MenuFlyoutItem>
    <RadioMenuFlyoutItem GroupName="Shipping">Premium shipping</RadioMenuFlyoutItem>
    <RadioMenuFlyoutItem GroupName="Shipping">Overnight shipping</RadioMenuFlyoutItem>
</RadioMenuFlyoutSubItem>
```

Reply:
Discussion: these two examples should work (separator and the label).

Recommend: make the recommendation more nuanced, or just a suggestion, etc.